### PR TITLE
Have a better default for port, one that can be overwritten

### PR DESCRIPTION
**Pull request by do3cc**

### CODE REVIEW

**Comment by do3cc** [comment]
Oooh, my local buildout only runs 34 tests...

**Comment by do3cc** [comment]
@saily please review and merge

File: collective/recipe/solrinstance/__init__.py [hunk 1]
```diff
@@ -399,13 +399,13 @@ def _split_index_line(self, line):
...
                     raise zc.buildout.UserError(
-                        'Invalid index definition: %s' % line)
+                        'Invalid index definition: {0}'.format(line))
                 params[len(params) - 1] += ' ' + each
             else:
                 params.append(each)
         return params
 
-    def get_indexes(self, options):
+    def get_indexes(self, options):  # NOQA
```
Comment (saily) [inline collective/recipe/solrinstance/__init__.py]: why do we need a `# NOQA` here?

Comment (do3cc) [inline collective/recipe/solrinstance/__init__.py]: Complexity check not passed

**Pull request merged**